service/src/main/java/com/shortlink/service/server/Base62LtS.java

```diff
@@ -15,9 +15,9 @@ private String base62(long num) {
     }
 
     public String encode(String url){
-        long numUrl = Long.parseLong(url);
+        long numUrl = url.split("https://www.")[1].split(".com")[0].hashCode();
         String result = base62(numUrl);
-        return  result;
+        return  "http://localhost:7000/stl/"+result;
```

Comment:
Put this in the properties instead of hard-code

Reply:
the endpoint is stored in application.properties